trace-encoder/api/trace-encoder.api

```diff
@@ -1,9 +1,79 @@
+public final class com/squareup/tracing/ChromeTraceEvent {
```

Comment:
I'm a bit confused as to why these wouldn't have been here before… seems like they should have been?

Reply:
I'd think so, too 🤷🏻

Reply:
Ok this wasn't here before because it is internal. Which means the fact that it's showing up now seems like a big.

Reply:
I just noticed we're a few versions behind on the validator plugin. Updating it in #419, testing locally to see if that fixes it…

Reply:
Yep that fixes everything!